cordova-lib/src/plugman/fetch.js

```diff
@@ -68,9 +68,13 @@ function fetchPlugin(plugin_src, plugins_dir, options) {
             if (result[2])
                 options.subdir = result[2];
             //if --fetch was used, throw error for subdirectories
-            if (result[2] && options.fetch) {
-                return Q.reject(new CordovaError('--fetch does not support subdirectories'));
-            }
+        
+        if(options.subdir) {
+            events.emit('warn', 'support for subdirectories is depreated and will be removed in Cordova@7');
+        if (options.fetch) {
```

Comment:
Probably an Indentation issue? This should has the same level of indentation as line above